packages/next/next-server/server/render.tsx

```diff
@@ -2,6 +2,7 @@ import { IncomingMessage, ServerResponse } from 'http'
 import { ParsedUrlQuery } from 'querystring'
 import React from 'react'
 import { renderToStaticMarkup, renderToString } from 'react-dom/server'
+import Critters from 'critters'
```

Comment:
If we do https://github.com/vercel/next.js/pull/16539/files#r520067116 we can move this to a conditional require